MinPlatformPkg/Test/Library/TestPointCheckLib/DxeTestPointCheckLib.c

```diff
@@ -815,6 +815,7 @@ TestPointReadyToBootMemoryTypeInformationFunctional (
     Result = FALSE;
   }
   TestPointDumpUefiMemoryMap (NULL, NULL, NULL, TRUE);
+  // Invalid memory type information reported here
```

Comment:
Can you elaborate?

Is this a follow up note for a specific platform? If so, you can remoove from here and track it against the platform. If it's a general comment that belongs here, please expand what it means.

Reply:
This was a comment specifically for Standalone MM that can't print out this data.  I've made changes for Standalone MM and I'll remove this comment.